test/main.cpp

```diff
@@ -1,4 +1,36 @@
 #include "ostream"
-
 #define CATCH_CONFIG_MAIN
 #include <catch.hpp>
+
+#include <test_usart.hpp>
+#include <uart_ports.hpp>
+
+TEST_CASE("test_usart_c sends","[test_usart_c]"){
```

Comment:
maybe a test with the ```set_receive_string()``` and the ```set_receive_bytes()```?

Reply:
were added in https://github.com/R2D2-2019/USART_library/pull/12/commits/85fdce96b4074228fcd84ff4c1db1e03da94d3f2